src/TheThingsNetwork.cpp

```diff
@@ -749,6 +749,39 @@ void TheThingsNetwork::configureKR920_923()
   sendMacSet(MAC_PWRIDX, TTN_PWRIDX_KR920_923);
 }
 
+void TheThingsNetwork::configureIN865_867()
+{
+  sendMacSet(MAC_ADR, "off"); // TODO: remove when ADR is implemented for this plan
+  sendMacSet(MAC_RX2, "2 866550000"); // SF10
+  
+  // Disable the three default LoRaWAN channels
+  sendChSet(MAC_CHANNEL_STATUS, 0, "off");
+  sendChSet(MAC_CHANNEL_STATUS, 1, "off");
+  sendChSet(MAC_CHANNEL_STATUS, 2, "off");
+
+  char buf[10];
+  uint32_t freq = 865062500;
+  uint8_t ch;
+  for (ch = 3; ch < 7; ch++)
+  {
+    sendChSet(MAC_CHANNEL_DCYCLE, ch, "799");
```

Comment:
For 1% duty cycle on 3 channels, this value should be `99`, not `799`.

```
The <dutyCycle> value that needs to be configured can be
obtained from the actual duty cycle X (in percentage)
using the following formula: <dutyCycle> = (100/X) – 1
```

Reply:
I shall try the same and update you soon.

Reply:
Actually now that I slept on this I doubt it again:

1% duty cycle over 3 channels is 0.333% per channel.
`(100/0.333) - 1 = 299`

So the value should be `299`.